src/core/exception_hacks.cc

```diff
@@ -73,13 +73,15 @@ module seastar;
 namespace seastar {
 using dl_iterate_fn = int (*) (int (*callback) (struct dl_phdr_info *info, size_t size, void *data), void *data);
 
-[[gnu::no_sanitize_address]]
+// once it was a lambda, [[gnu::no_sanitize_address]] did not help in debug.
```

Comment:
So now that it's not a lambda, you don't need it any more? I don't understand.

Reply:
> I have to be honest, I didn't understand what bug this patch is fixing

Its fixing a segfault at static lambda eval point in Debug build with ASAN for Seastar hello-world demo and all other apps.

> and why we haven't seen it in the past. 

I have no idea.

> Does it happen on a specific compiler? In an application doing something unusual?

no. Its arch linux, gcc, all your demo apps.